sdk/communication/communication-jobrouter/swagger/swagger.json

```diff
@@ -0,0 +1,3452 @@
+{
+  "swagger": "2.0",
```

Comment:
Why do we have this swagger file here instead of referencing it from the specs repo?